### ISSUE TITLE
Nnl_NL

### ISSUE THREAD

**Issue by Sitethief** [fzaninotto/Faker#1801]
### Summary

When calling `$faker->titleMale`  with the locale set to `'nl_NL'` an exception is thrown.

`ErrorException : Non-static method Faker\Provider\nl_NL\Person::title() should not be called statically`.

I'm not allowed to set `title` to be static though.

### Versions

|                     | Version |
|:--------------------|:--------|
| PHP                 |  7.2.16   |
| `fzaninotto/faker`  | v1.8.0 |
| Laravel | v5.5 |

### Self-enclosed code snippet for reproduction

```php
echo $faker->titleMale;
```

### Expected output

```txt
'Dhr'
```

### Actual output

```txt
ErrorException : Non-static method Faker\Provider\nl_NL\Person::title() should not be called statically
 /var/www/vendor/fzaninotto/faker/src/Faker/Provider/nl_NL/Person.php:291
 /var/www/vendor/fzaninotto/faker/src/Faker/Generator.php:222
 /var/www/vendor/fzaninotto/faker/src/Faker/Generator.php:268
```

**Comment by pimjansen** [comment]
This is a valid error since the method is not available. There is no available method to access only male titles for this locale. These are the ones that are available:

```php
    protected static $title = array(
        'mr.', 'dr.', 'ir.', 'drs', 'bacc.', 'kand.', 'dr.h.c.', 'prof.', 'ds.', 'ing.', 'bc.'
    );
````

Feel free to PR this with the split between male and female.If not possible let me know so i can help

**Comment by erikgaal** [comment]
This has been fixed in #1696, but is not yet released.

**Comment by pimjansen** [comment]
Thanks @erikgaal!

It will be release in 1.9.0 which will be there probably mid nov.
